library/socket/unixserver/new_spec.rb

```diff
@@ -2,5 +2,13 @@
 require_relative 'shared/new'
 
 describe "UNIXServer.new" do
-  it_behaves_like :unixserver_new, :new
+  platform_is_not :windows do
```

Comment:
Could you move the `platform_is_not` outside? Because UNIXServer is not even defined on Windows (actually it is in some cases but we are ignoring that so far to keep things simple). Same below.

Reply:
I extended this one a little bit to keep things consistent:
* Replace `platform_is_not :windows` with `with_feature :unix_socket`, which looks a bit more generic. The current code had both in use. This could resolve the "in some cases on Windows", dependign on the implementation of `with_feature`.
* Move these guards to the outer block in all spec files, and remove them from the shared files.

Reply:
Great, that's even better, thank you!